crates/exex/exex/src/manager.rs

```diff
@@ -139,6 +153,78 @@ impl ExExHandle {
     }
 }
 
+/// A subscriber for [`ExExNotifications`].
+#[derive(Debug)]
+pub struct ExExNotificationsSubscriber {
+    notifications: ExExNotifications,
+    state_tx: watch::Sender<ExExNotificationsState>,
```

Comment:
here the channel already exists?

If I understand this correctly then the channel is established but requires a `ExExNotificationsState` update to make it active?

this feels a bit weird, 
I think we'd like more idiomatic would be some kind of command channel that receives the appropriate notification channel as return value, like `pub fn subscribe_with_head(` but then notification subscription is created on demand?